server/svix-server/migrations/20220428190104_message_expiration.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+ALTER TABLE message ADD COLUMN expiration timestamp with time zone;
+ALTER TABLE message ALTER COLUMN payload DROP NOT NULL;
+CREATE INDEX ON message (payload) WHERE payload IS NOT NULL;
```

Comment:
1. Explicitly set a name of the index (see in other SQL files how it's done).
2. The index should be on `expiration` not on `payload`.

Reply:
1. 👍 
2. Perhaps this index should be on both columns? Originally, `payload` was the column of concern, but since adding this business logic migration, `expiration` is as well as pointed out here.

Reply:
Still wrong, it should be:
```
CREATE INDEX ON message (expiration) WHERE payload IS NOT NULL; 
```